components/about-us/sub-pages/projects/content/governator.tsx

```diff
@@ -0,0 +1,87 @@
+import { Box, Text, Stack, Flex, Image, Heading, Link } from '@chakra-ui/react'
+import { FiArrowUpRight } from 'react-icons/fi'
+
+export default function GovernatorComponent() {
+  return (
+    <Stack
+      justify='space-between'
+      align={'center'}
+      width={{ md: '750px', lg: '800px', xl: '1000px' }}
+      pt={{ base: 10, md: 14 }}
+      direction={{ base: 'column', md: 'row' }}
+    >
+      <Flex
+        flex={1}
+        justify={'flex-start'}
+        alignItems={'center'}
+        position={'relative'}
+        w={'full'}
+      >
+        <Box
+          position={'relative'}
+          height={{ base: '300px', sm: '350px' }}
+          width={{ base: '300px', sm: '350px' }}
+          boxShadow={'2xl'}
+          overflow={'hidden'}
+        >
+          <Image
+            alt={'Bankless logo'}
+            fit={'cover'}
+            align={'center'}
+            w={'100%'}
+            h={'100%'}
+            src='/images/gov-bot.jpeg'
+          />
+        </Box>
+      </Flex>
+
+      <Stack
+        width={{ base: '300px', sm: '350px', xl: '500px' }}
+        spacing={{ base: 5, md: 4 }}
+      >
+        <Heading
+          color={'gray.100'}
+          width={{ base: '300px', sm: '350px', xl: '500px' }}
+          fontWeight={700}
+          mt={{ base: '32px', md: '0' }}
+          fontSize={{ base: '20px' }}
+        >
+          GOVERNATOR
+        </Heading>
+        <Text
+          color={'gray.100'}
+          width={{ base: '300px', sm: '350px', xl: '500px' }}
+          lineHeight={{
+            base: '20px',
+            sm: '27px',
+            md: '30px',
+            lg: '30px',
+            xl: '45px',
+          }}
+          fontSize={{ base: '16px', sm: '20px' }}
+        >
+          Token Voting in Discord Easy to use, token-enabled, shielded polls
+          right in your server.
```

Comment:
Minor grammatical errors:

```suggestion
          Token voting in Discord. Easy-to-use, token-enabled, shielded polls
          right in your server.
```

Reply:
Fixed by https://github.com/BanklessDAO/bankless-website/pull/198